pkg/bosh/manifest/container_factory.go

```diff
@@ -212,9 +213,11 @@ func jobSpecCopierContainer(releaseName string, jobImage string, volumeMountName
 			},
 		},
 		Command: []string{
-			"bash",
-			"-c",
-			fmt.Sprintf(`mkdir -p "%s" && cp -ar %s/* "%s"`, inContainerReleasePath, VolumeJobsSrcDirMountPath, inContainerReleasePath),
+			"/bin/sh",
+		},
+		Args: []string{
+			"-xc",
+			specCopierCmd,
```

Comment:
Why a new variable?

Reply:
when I changed this, I was thinking on improving test cases, by having a fixed command constant that can be used in the code and test cases. I think this cannot be a constant and I forgot to rollback this change, will do it now.